tests/integration/test_init.py

```diff
@@ -54,7 +54,10 @@ def setup_postgres(postgres_service):
 
 @pytest.fixture
 def setup_dbt(setup_postgres):
-    subprocess.run(["dbt", "clean", *DBT_ARGS], check=True)
+    with pytest.MonkeyPatch.context() as mp:
+        mp.setenv('DBT_CLEAN_PROJECT_FILES_ONLY', 'false')
+        subprocess.run(["dbt", "clean", *DBT_ARGS], check=True)
+    
```

Comment:
Can you please revert this and rather include the environment variable in the `tox.ini`? I'm just trying to get rid of the monkey-patching.

```suggestion
    subprocess.run(["dbt", "clean", *DBT_ARGS], check=True)
```